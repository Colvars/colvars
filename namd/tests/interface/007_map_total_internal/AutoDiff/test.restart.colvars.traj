# step         numatoms               numatoms_rot           numatoms_ref          
          20    5.06628721207380e+01   1.99557660952032e+01   5.06628721207380e+01 
          21    5.06524273268878e+01   1.99296087794555e+01   5.06524273268878e+01 
          22    5.06455445662141e+01   1.99059675278786e+01   5.06455445662141e+01 
          23    5.06419908143580e+01   1.98843517200421e+01   5.06419908143580e+01 
          24    5.06406837888062e+01   1.98640745095145e+01   5.06406837888062e+01 
          25    5.06395369693637e+01   1.98440915231467e+01   5.06395369693637e+01 
          26    5.06362216956913e+01   1.98257476314202e+01   5.06362216956913e+01 
          27    5.06286801546812e+01   1.98048617019701e+01   5.06286801546812e+01 
          28    5.06148089617491e+01   1.97839549465107e+01   5.06148089617491e+01 
          29    5.05931192599237e+01   1.97637992634232e+01   5.05931192599237e+01 
          30    5.05629946291447e+01   1.97452153663512e+01   5.05629946291447e+01 
          31    5.05247787106782e+01   1.97287612106265e+01   5.05247787106782e+01 
          32    5.04798152558506e+01   1.97145576721301e+01   5.04798152558506e+01 
          33    5.04303130898625e+01   1.97020908897907e+01   5.04303130898625e+01 
          34    5.03789391908795e+01   1.96905407908171e+01   5.03789391908795e+01 
          35    5.03285725582391e+01   1.96790245122938e+01   5.03285725582391e+01 
          36    5.02819653376937e+01   1.96668252900599e+01   5.02819653376937e+01 
          37    5.02413368560374e+01   1.96538983686878e+01   5.02413368560374e+01 
          38    5.02084177769721e+01   1.96409041557117e+01   5.02084177769721e+01 
          39    5.01839407384396e+01   1.96289455688730e+01   5.01839407384396e+01 
          40    5.01673023067415e+01   1.96190860426341e+01   5.01673023067415e+01 
