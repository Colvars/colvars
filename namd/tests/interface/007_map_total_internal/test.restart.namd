# -*- tcl -*-

set output_name "test.restart"
set coor_bin_file "test.coor"
set vel_bin_file "test.vel"

source ../Common/common.namd
source ../Common/measure_net_force_torque.tcl

colvars on

source gridforcemaps.tcl

cv configfile test.in

cv load test.colvars.state

run ${numsteps}

source ../Common/delete_tmp_files.tcl
