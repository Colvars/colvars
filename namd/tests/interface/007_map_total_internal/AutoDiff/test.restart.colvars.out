colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: This version was built with the C++11 standard or higher.
colvars: Summary of compile-time features available in this build:
colvars:   - SMP parallelism: not available
colvars:   - Lepton custom functions: available
colvars:   - Tcl interpreter: available
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # indexFile = "../Common/da.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # sourceTclFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "numatoms"
colvars:   Initializing a new "mapTotal" component.
colvars:     # name = "numatoms"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # mapName = "" [default]
colvars:     # mapID = 0
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "Protein"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 104 atoms requested: total mass = 727.821, total charge = 6.70552e-08.
colvars:     # atomWeights =  [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 2
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 2 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "numatoms_rot"
colvars:   Initializing a new "mapTotal" component.
colvars:     # name = "numatoms_rot"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # mapName = "" [default]
colvars:     # mapID = 0
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = on
colvars:       # rotateToReference = on
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "Protein"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       Within atom group "atoms":
colvars:       Initializing atom group "fittingGroup".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "fittingGroup" defined with 5 atoms requested: total mass = 60.055, total charge = 0.35.
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = "../Common/310turn-ideal.xyz"
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: beginning from 2019-11-26 the XYZ file reader assumes Angstrom units.
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use fittingGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 104 atoms requested: total mass = 727.821, total charge = 6.70552e-08.
colvars:     # atomWeights =  [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 2
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 2 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "numatoms_ref"
colvars:   Initializing a new "mapTotal" component.
colvars:     # name = "numatoms_ref"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # mapName = "map"
colvars:     # mapID = -1 [default]
colvars:     # atomWeights =  [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 2
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 2 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 3 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { numatoms }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 50 }
colvars:   # targetCenters = { 50 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 0.2
colvars:   # decoupling = off [default]
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "numatoms" will be rescaled to 0.05 according to the specified width (2).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (104 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (1 grid objects in total).
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - Colvars module:
colvars: - Colvars-NAMD interface:
colvars: - Harmonic colvar bias implementation:
colvars: - Moving frame of reference:
colvars: - Optimal rotation via flexible fitting:
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - Volumetric map-based collective variables:
colvars: - mapTotal colvar component:
colvars:   Fiorin2020 https://doi.org/10.1002/jcc.26075
colvars: 
colvars: - NAMD engine:
colvars:   Phillips2020 https://doi.org/10.1063/5.0014475
colvars: 
colvars: - GridForces volumetric map implementation for NAMD:
colvars:   Wells2007 https://doi.org/10.1063/1.2770738
colvars: 
colvars: updating target temperature (T = 0 K).
colvars: ----------------------------------------------------------------------
colvars: Loading state from text file "test.colvars.state".
colvars:   Restarting collective variable "numatoms" from value: 50.6627
colvars:   Restarting collective variable "numatoms_rot" from value: 19.9852
colvars:   Restarting collective variable "numatoms_ref" from value: 50.6627
colvars:   Restarted harmonic bias "harmonic1" with step number 20.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (104 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (1 grid objects in total).
colvars: updating target temperature (T = 0 K).
colvars: Current simulation parameters: initial step = 20, integration timestep = 1
colvars: Updating atomic parameters (masses, charges, etc).
colvars: Re-initialized atom group for variable "numatoms":0/0. 104 atoms: total mass = 727.821, total charge = 6.70552e-08.
colvars: Re-initialized atom group for variable "numatoms_rot":0/0. 104 atoms: total mass = 727.821, total charge = 6.70552e-08.
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Saving collective variables state to "test.restart.colvars.state".
